client/src/CodeClaimPage/templateClaim/TemplateClaimFooter.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC } from 'react';
+
+// assets
+import PoapBadge from '../../images/POAP.svg';
+import Twitter from '../../images/logo-twitter.svg';
+import Telegram from '../../images/logo-telegram.svg';
+import Github from '../../images/logo-git.svg';
+
+// types
+import { TemplatePageFormValues } from '../../templates/TemplateFormPage/components/TemplateForm';
+import { HashClaim } from '../../api';
+import { useImageSrc } from '../../lib/hooks/useImageSrc';
+
+type Props = {
+  template?: TemplatePageFormValues;
+  claim?: HashClaim;
+};
+
+export const TemplateClaimFooter: FC<Props> = ({ template, claim }) => {
+  const templateFooterIconRaw = claim?.event_template?.footer_icon
+    ? claim?.event_template?.footer_icon
+    : template?.footer_icon;
+  const templateFooterColor = claim?.event_template?.footer_color
+    ? claim?.event_template?.footer_color
+    : template?.footer_color;
```

Comment:
Prefer nullish coalescing operator over ternary operator for this cases

```suggestion
  const templateFooterIconRaw = claim?.event_template?.footer_icon ?? template?.footer_icon;
  const templateFooterColor = claim?.event_template?.footer_color ?? template?.footer_color;
```